src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -170,6 +170,11 @@ public function get(): ?array
         } catch (\Throwable $e) {
             $this->driverConnection->rollBack();
 
+            if ($this->autoSetup && $e instanceof TableNotFoundException) {
+                $this->setup();
+                goto get;
```

Comment:
I'm sure people will get crazy about this line :)

Reply:
did it intentionally :)